_includes/user-menu.html

```diff
@@ -1,101 +1,130 @@
-      <li><a href="{{docu}}/installation/synology.html">Synology</a></li>
-      <li><a href="{{docu}}/installation/designer.html">openHAB Designer</a></li>
-    </ul>
-  </li>
-  <li><a href="{{docu}}/configuration/nginx.html">Configuration</a>
-    <ul>
-      <li><a href="{{docu}}/configuration/nginx.html">Reverse Proxy</a></li>
-    </ul>
-  </li>
-  <li><a href="{{docu}}/administration/index.html">Administration</a>
-    <ul>
-      <li><a href="{{docu}}/administration/index.html">Overview</a></li>
-      <li><a href="{{docu}}/administration/console.html">Console</a></li>
-      <li><a href="{{docu}}/administration/logging.html">Logging</a></li>
-      <li><a href="{{docu}}/administration/bundles.html">Bundle Management</a></li>
-      <li><a href="{{docu}}/administration/runtime.html">Runtime Commands</a></li>
+      <li><a href="{{docu}}/installation/synology.html">Synology DiskStation</a></li>
+      <li><a href="{{docu}}/installation/qnap.html">QNAP NAS</a></li>
+      <hr />
+      <li><a href="{{docu}}/installation/designer.html">SmartHome Designer</a></li>
+      <li><a href="{{docu}}/installation/nginx.html">Authentication &amp; HTTPS</a></li>
+      <li><a href="https://my.openhab.org/docs">my.openHAB (external)</a></li>
     </ul>
   </li>
-  <li><a href="{{docu}}/features/index.html">Features</a>
+  <li><a href="{{docu}}/configuration/index.html">Configuration</a>
     <ul>
-      <li><a href="{{docu}}/features/index.html">Overview</a></li>
-      <li><a href="{{docu}}/features/items.html">Items</a></li>
-      <li><a href="{{docu}}/features/sitemap.html">Sitemaps</a></li>
-      <li><a href="{{docu}}/features/persistence.html">Persistence</a></li>
-      <li><a href="{{docu}}/features/multimedia.html">Multimedia</a></li>
-      <li><a href="{{docu}}/features/automation/ruledsl.html">Automation</a>
-        <ul>
-          <li><a href="{{docu}}/features/automation/ruledsl.html">Textual Rules</a></li>
-          <li><a href="{{docu}}/features/automation/ngrules.html">Next-Gen Rules</a></li>
-        </ul>
-      </li>
+      <li><a href="{{docu}}/configuration/index.html">Overview</a></li>
+      <hr />
+      <li><a href="{{docu}}/configuration/things.html">Things</a></li>
+      <li><a href="{{docu}}/configuration/items.html">Items</a></li>
+      <li><a href="{{docu}}/configuration/sitemaps.html">Sitemaps</a></li>
+      <li><a href="{{docu}}/configuration/transform.html">Transformations</a></li>
+      <li><a href="{{docu}}/configuration/persistence.html">Persistence</a></li>
+      <li><a href="{{docu}}/configuration/rules-dsl.html">Rules</a></li>
+      <li><a href="{{docu}}/configuration/services.html">Services</a></li>
+      <hr />
+      <li><a href="{{docu}}/configuration/packages.html">Initial Setup Packages</a></li>
+      <li><a href="{{docu}}/configuration/ui-based.html">Paper UI Discovery</a></li>
+      <li><a href="{{docu}}/configuration/ui-based.html">Paper UI Configuration</a></li>
```

Comment:
having 4 entries with the same url results in all of them being selected in the menu:

<img width="206" alt="screen shot 2016-12-12 at 15 22 26" src="https://cloud.githubusercontent.com/assets/3244965/21102736/ad4b30a4-c07f-11e6-9327-fe865a74e2ef.png">

Is this intended?